src/components/Checkbox/__tests__/Checkbox.test.tsx

```diff
@@ -0,0 +1,192 @@
+import React from 'react';
+import userEvent from '@testing-library/user-event';
+import {render, screen} from '@testing-library/react';
+import {Checkbox, CheckboxSize} from '../Checkbox';
+
+const qaId = 'checkbox-component';
+
+describe('Checkbox', () => {
+    test('render checkbox by default', () => {
+        render(<Checkbox />);
+        const checkbox = screen.getByRole('checkbox');
+
+        expect(checkbox).toBeVisible();
+        expect(checkbox).not.toBeDisabled();
+        expect(checkbox).not.toBeChecked();
+    });
+
+    test.each(new Array<CheckboxSize>('m', 'l'))('render with given "%s" size', (size) => {
+        render(<Checkbox size={size} qa={qaId} />);
+        const component = screen.getByTestId(qaId);
```

Comment:
This component got appropriate `label` role, do we really need qa attributes here?

Reply:
`label` element has no corresponding role. [See w3 roles table](https://www.w3.org/TR/html-aria/#docconformance)